### PR TITLE
Update some of hardcoded perl.com links

### PR DESCRIPTION
**Pull request by kyzn**
Hi! I found some more hardcoded perl.com links. This PR comes in two commits, one replaces two instances with relative "/" in legacy content. Other one puts hardcoded perldotcom.perl.org in places like GitHub readme and DDG search.

I've also found two "training.perl.com", but wasn't sure what to do so I left those as is.

Hope you find this useful. Do let me know if you need any updates or clarification. Thanks!

### CODE REVIEW

File: content/legacy/_pub_2001_08_13_yapc-europe.md [hunk 1]
```diff
@@ -52,4 +52,4 @@ The second edition of Yet Another Perl Conference in Europe was a great success.
 
 ------------------------------------------------------------------------
 
-Return to [Perl.com](http://perl.com)
+Return to [Perl.com](/)
```
Comment (briandfoy) [inline content/legacy/_pub_2001_08_13_yapc-europe.md:55]: Let's just delete that line and remove the `---` above it. I made similar changes to another article last night.

File: bin/site-is-fresh [hunk 2]
```diff
@@ -30,7 +30,7 @@ the live site is fresh.
 use IO::Interactive qw(interactive);
 use Mojo::UserAgent;
 
-my $url = 'https://www.perl.com/json/build.json';
```
Comment (briandfoy) [inline bin/site-is-fresh:33]: Don't change this file. This is mostly something I use.

File: Makefile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-SITE=https://www.perl.com
+SITE=https://perldotcom.perl.org
```
Comment (briandfoy) [inline Makefile:1]: We don't need to change this one just yet. This simply supports a tool that I personally use and I already have other measures in place.

**Review by briandfoy — CHANGES_REQUESTED** [review]
A few minor adjustments and this will be good to go. Thanks!

**Comment by kyzn** [comment]
I've updated the PR. Removed the search part since that's already on master.

**Review by briandfoy — APPROVED** [review]

**Comment by briandfoy** [comment]
Thanks for the work!

**Pull request merged**